packages/strapi-admin/admin/src/containers/Admin/index.js

```diff
@@ -0,0 +1,386 @@
+/**
+ *
+ * Admin
+ *
+ */
+
+import React from 'react';
+import ReactGA from 'react-ga';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+import { bindActionCreators, compose } from 'redux';
+import { Switch, Route } from 'react-router-dom';
+
+// Actions from strapi-helper-plugin
+// Actions required for disabling and enabling the OverlayBlocker
+import {
+  disableGlobalOverlayBlocker,
+  enableGlobalOverlayBlocker,
+} from 'actions/overlayBlocker';
+
+// Components from strapi-helper-plugin
+import LoadingIndicatorPage from 'components/LoadingIndicatorPage';
+import OverlayBlocker from 'components/OverlayBlocker';
+
+import injectHooks from 'utils/injectHooks';
+
+import Header from '../../components/Header/index';
+import Logout from '../../components/Logout';
+
+import ComingSoonPage from '../ComingSoonPage';
+import LeftMenu from '../LeftMenu';
+import LocaleToggle from '../LocaleToggle';
+import HomePage from '../HomePage/Loadable';
+import Marketplace from '../Marketplace/Loadable';
+import ListPluginsPage from '../ListPluginsPage/Loadable';
+import NotFoundPage from '../NotFoundPage/Loadable';
+import PluginDispatcher from '../PluginDispatcher';
+
+import { updatePlugin } from '../App/actions';
+import makeSelecApp from '../App/selectors';
+
+import injectSaga from '../../utils/injectSaga';
+import injectReducer from '../../utils/injectReducer';
+
+import localeToggleReducer from '../LocaleToggle/reducer';
+import {
+  resetLocaleDefaultClassName,
+  setLocaleCustomClassName,
+} from '../LocaleToggle/actions';
+
+import {
+  getInitData,
+  getSecuredData,
+  hideLeftMenu,
+  hideLogout,
+  setAppError,
+  setAppSecured,
+  showLeftMenu,
+  showLogout,
+  unsetAppSecured,
+} from './actions';
+import makeSelectAdmin from './selectors';
+import reducer from './reducer';
+import saga from './saga';
+
+import NavTopRightWrapper from './NavTopRightWrapper';
+
+import styles from './styles.scss';
+
+export class Admin extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  state = { shouldSecureAfterAllPluginsAreMounted: true };
+
+  getChildContext = () => ({
+    currentEnvironment: this.props.admin.currentEnvironment,
+    disableGlobalOverlayBlocker: this.props.disableGlobalOverlayBlocker,
+    enableGlobalOverlayBlocker: this.props.enableGlobalOverlayBlocker,
+    plugins: this.props.global.plugins,
+    updatePlugin: this.props.updatePlugin,
+  });
+
+  componentDidMount() {
+    // Initialize Google Analytics
+    // Refer to ../../../doc/disable-tracking.md for more informations
+    /* istanbul ignore next */
+    ReactGA.initialize(
+      'UA-54313258-9',
+      { testMode: process.env.NODE_ENV === 'test' },
```

Comment:
This should another variable because it messes with the currentEnvironment so some part of the UI aren't rendered if we have the webpack server and the tests running at the same time.